posthog/settings.py

```diff
@@ -733,9 +733,4 @@ def show_toolbar(request):
 # keep in sync with plugin-server
 EVENTS_DEAD_LETTER_QUEUE_STATSD_METRIC = "events_added_to_dead_letter_queue"
 
-VERSION_REQUIREMENTS = [VersionRequirement(service="postgresql", supported_version=">=12.0.0,<15.0.0",)]
-
-if PRIMARY_DB == AnalyticsDBMS.CLICKHOUSE:
-    VERSION_REQUIREMENTS = VERSION_REQUIREMENTS + [
-        VersionRequirement(service="clickhouse", supported_version=">=21.6.0,<21.7.0"),
-    ]
+VERSION_REQUIREMENTS = [VersionRequirement(service="postgresql", supported_version=">=12.0.0,<15.0.0")]
```

Comment:
Should we add Redis for the default version?

Reply:
added